src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -906,21 +908,19 @@ else if (v instanceof ListValue)
             }
         });
 
-        put("silent",(e,v)->{
-           e.setSilent(v.getBoolean());
-        });
+        put("silent",(e,v)-> e.setSilent(v.getBoolean()));
 
-        put("gravity",(e,v)->{
-            e.setNoGravity(!v.getBoolean());
-        });
+        put("gravity",(e,v)-> e.setNoGravity(!v.getBoolean()));
 
-        put("invulnerable",(e,v)->{
-            e.setInvulnerable(v.getBoolean());
-        });
+        put("invulnerable",(e,v)-> e.setInvulnerable(v.getBoolean()));
 
-        put("fire",(e,v)->{
-            e.setFireTicks((int)NumericValue.asNumber(v).getLong());
-        });
+        put("fire",(e,v)-> e.setFireTicks((int)NumericValue.asNumber(v).getLong()));
+
+        put("hunger", (e, v)-> ((PlayerEntity) e).getHungerManager().setFoodLevel((int)NumericValue.asNumber(v).getLong()));
```

Comment:
As above, this should be expanded to something like this (including removing the extra parenthesis):
```suggestion
        put("hunger", (e, v)-> {
            ((PlayerEntity) e).getHungerManager().setFoodLevel((int)NumericValue.asNumber(v).getLong());
        }
```
(same for next 2 occurrences)

Reply:
yah, no

Reply:
I don't think it's necessary, and it bloats file unnecessarily, and I think it's better to look at this way.

Reply:
I am fine either way.

best way - not changing anything. This code lives in two mc versions at a time (1.16) so better not change / refactor unnecessarily code since it gets messy when merging with 1.16 branch

Reply:
so what, should I change or no change?